store-api-server/migrations/1660154421-simplex-payment-provider.sql

```diff
@@ -0,0 +1,16 @@
+-- Migration: simplex-payment-provider
+-- Created at: 2022-08-10 21:00:21
+-- ====  UP  ====
+
+BEGIN;
+
+ALTER TYPE payment_provider ADD VALUE IF NOT EXISTS 'simplex';
```

Comment:
This is good now but there is still the change in the old migration file, that should still be undone.